src/k8s/pkg/k8sd/features/coredns/chart.go

```diff
@@ -11,12 +11,12 @@ var (
 	Chart = helm.InstallableChart{
 		Name:         "ck-dns",
 		Namespace:    "kube-system",
-		ManifestPath: filepath.Join("charts", "coredns-1.29.0.tgz"),
+		ManifestPath: filepath.Join("charts", "coredns-1.36.0.tgz"),
 	}
 
 	// imageRepo is the image to use for CoreDNS.
 	imageRepo = "ghcr.io/canonical/coredns"
 
 	// ImageTag is the tag to use for the CoreDNS image.
-	ImageTag = "1.11.1-ck4"
+	ImageTag = "1.11.3"
```

Comment:
Something for later: Maybe we can use resolve these from the actual binaries. Having these versions and numbers duplicated a multiple places scares me...
Created a card for it.